packages/functional-tests/tests/subscription-tests/subscription.spec.ts

```diff
@@ -177,6 +177,7 @@ test.describe('severity-2 #smoke', () => {
         expect(page.url()).toContain('&flow_begin_time=');
       });
 
+      /* Disabling test - to fix in FXA-8447
```

Comment:
@ashrivastava-qa Not sure if you want to do a followup PR to convert to [`.fixme()`](https://playwright.dev/docs/api/class-test#test-fixme-2) or [`.skip()`](https://playwright.dev/docs/api/class-test#test-skip-2) annotations instead so these are easier to track and reenable in the future.
https://playwright.dev/docs/test-annotations#skip-a-test